simple_history/models.py

```diff
@@ -133,6 +133,34 @@ def finalize(self, sender, **kwargs):
         setattr(sender, self.manager_name, descriptor)
         sender._meta.simple_history_manager_attribute = self.manager_name
 
+    def check_custom_model_name_unique(self, model):
+        # if the custom model name is the same as the original model name,
+        # make sure that the app names are different
+        try:
```

Comment:
Weird try block. Why isn't this just 
```
if not self.app or self.app == model._meta.app_label:
    raise ValueError( "The 'custom_model_name'....)
```

Reply:
I agree but saw it done that way in the __init__ method and assumed it might be a convention used in your app so did it the same ... will change.